modules/standard/Random.chpl

```diff
@@ -915,25 +1124,87 @@ module Random {
 
       /* Generate a random number in [0,bound).
 
+         This function corresponds to pcg32_boundedrand_r and can
+         call the random-number generator more than once.
+
+         :arg inc: The sequence constant (same as passed to `srandom`)
+         :arg bound: The returned value will be < `bound`.
+         :returns: a random number in [0,bound).
+       */
+      inline
```

Comment:
Just because it's short. Although it looks like a few function calls, it's actually maybe 8 instructions.